lib/rexport/data_fields.rb

```diff
@@ -106,8 +104,8 @@ def get_rexport_method(field_name)
         end
       end
 
-      def reset_column_information_with_rexport_reset
-        reset_column_information_without_rexport_reset
+      def reset_column_information
+        super
```

Comment:
@davidgross, put the super call at the end so that is the return value from this method